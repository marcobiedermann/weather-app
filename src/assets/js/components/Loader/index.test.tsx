import React from 'react';
import renderer from 'react-test-renderer';
import Loader from '.';

describe('components/Loader', () => {
  it('renders correctly', () => {
    expect.assertions(1);

    const tree = renderer.create(<Loader />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
