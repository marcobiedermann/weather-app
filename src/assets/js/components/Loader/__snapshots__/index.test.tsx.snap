// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Loader renders correctly 1`] = `
<div>
  loading â€¦
</div>
`;
